Packages/MIES/MIES_Utilities.ipf

```diff
@@ -4534,6 +4534,12 @@ threadsafe Function/S num2strHighPrec(val, [precision])
 	return str
 End
 
+/// @brief Round the given number to the given number of decimal digits
+threadsafe Function RoundNumber(variable val, variable precision)
+
+	return str2num(num2strHighPrec(val, precision = precision))
```

Comment:
@t-b why does rounding require converting a variable to a string (and back)?

Reply:
@timjarsky This is the easiest approach code wise, it might not be the fastest though. We don't want to do the rounding ourselves therefore we let sprintf in num2strHighprec do it.